lib/matplotlib/backends/backend_webagg.py

```diff
@@ -20,31 +20,14 @@
 import signal
 import threading
 
-try:
-    import tornado
-except ImportError as err:
-    raise RuntimeError("The WebAgg backend requires Tornado.") from err
-
-import tornado.web
-import tornado.ioloop
-import tornado.websocket
+from js import document
```

Comment:
We don't support matplotlib in Node.js anyways, but importing document in Node.js or other Node.js-like environments will fail.